policies/broken-nightlies.md

```diff
@@ -0,0 +1,30 @@
+# Policy on broken nightlies
+
+Sometimes the nightlies released automatically by our CI ends up being broken
+for some people or even everyone. This policy defines what the infra team
+response will be in those cases.
+
+## Which nightly will be rolled back
+
+* A nightly can be rolled back if it contains destructive code, for
```

Comment:
Should this say "can" or "will"?  Sounds like the extreme case where it really should be rolled back.

Reply:
Well, sure, that will happen, but I'd like for this policy not to be binding.

Reply:
Well it's not _really_, is it?

Reply:
*shrugs* the wording is not really much of a problem